tests/conftest.py

```diff
@@ -87,54 +89,30 @@ def geom_geojson():
     # these need to be tuples, not list, or they will be changed
     # by shapely
     return {
-        "type": "Polygon",
-        "coordinates": [[
-            [
-                37.791595458984375,
-                14.84923123791421
-                ],
-            [
-                37.90214538574219,
-                14.84923123791421
-                ],
-            [
-                37.90214538574219,
-                14.945448293647944
-                ],
-            [
-                37.791595458984375,
-                14.945448293647944
-                ],
-            [
-                37.791595458984375,
-                14.84923123791421
-                ]
-        ]]
-      }
+        "type":
+        "Polygon",
+        "coordinates":
+        [[[37.791595458984375, 14.84923123791421],
+          [37.90214538574219, 14.84923123791421],
+          [37.90214538574219, 14.945448293647944],
+          [37.791595458984375, 14.945448293647944],
+          [37.791595458984375, 14.84923123791421]]]
+    }  # yapf: disable
```

Comment:
Yapf kind mangles this so I just disable it here.

Reply:
Yeah, there's nothing to do about readability of GeoJSON arrays :rofl: